src/command.ts

```diff
@@ -0,0 +1,261 @@
+import { first, last, get, size, head, keys, values } from "lodash";
+import { bold } from "cli-color";
+
+import { FirebaseError } from "./error";
+import { getInheritedOption } from "./utils";
+import { load } from "./rc";
+import { load as _load } from "./config";
+import * as configstore from "./configstore";
+import detectProjectRoot = require("./detectProjectRoot");
+import logger = require("./logger");
+import track = require("./track");
+import { CommanderStatic } from "commander";
+
+type ActionFunction = (...args: any[]) => Promise<any> | any;
```

Comment:
What's up with `Promise<any> | any`?  Isn't that kinda like `value || true`?

Reply:
ooo - I can get rid of that. I was fighting with the compiler for a bit and ended up with that, but I can consolidate this to `any`.